sys/include/net/gcoap.h

```diff
@@ -633,6 +633,20 @@ size_t gcoap_obs_send(const uint8_t *buf, size_t len,
  */
 uint8_t gcoap_op_state(void);
 
+/**
+ * @brief   Add Uri-Query options to a CoAP request
```

Comment:
I suggest "Adds a Uri-Query option..." I have used the 3rd person verb rather than the 2nd person imperative form for other gcoap commands.